src/menu/charprofile.c

```diff
@@ -255,6 +240,15 @@ static void charprofile_draw(MenuData *m) {
 	if(selected != PROFILE_LOCKED) {
 		portrait_params.sprite_ptr = portrait_get_face_sprite(profiles[selected].name, profiles[selected].faces[ctx->face]);
 		r_draw_sprite(&portrait_params);
+		r_mat_mv_push();
+		text_draw_wrapped("Press [Fire] for alternate expressions", DESCRIPTION_WIDTH, &(TextParams) {
+			.align = ALIGN_LEFT,
+			.pos = { 25, 570 },
+			.font = "standard",
+			.shader = "text_default",
+			.color = RGBA(0.9, 0.9, 0.9, 0.9),
+		});
+		r_mat_mv_pop();
```

Comment:
No need for `r_mat_mv_push()`/`r_mat_mv_pop()` here since you aren't modifying the matrix.